js/networking/request.js

```diff
@@ -0,0 +1,75 @@
+/*
+Class with methods for creating requests
+*/
+
+// http request methods
+export const methods = {
+  GET: 'GET',
+  POST: 'POST',
+  PATCH: 'PATCH',
+  DELETE: 'DELETE',
+  // HEAD: 'HEAD',
+  // PUT: 'PUT',
+  // OPTIONS: 'OPTIONS',
+};
+
+// Helper method creting 'method' request
+export function createRequest(method) {
+  return {
+    method,
+    headers: {
+      Accept: 'application/json',
```

Comment:
`application/vnd.api+json` is the proper `Accept` type, the API is not in this regard 100%, but thats the correct Accept header :)